runtime/src/iree/hal/string_util.c

```diff
@@ -164,6 +177,36 @@ IREE_API_EXPORT iree_status_t iree_hal_parse_element_type(
 IREE_API_EXPORT iree_status_t iree_hal_format_element_type(
     iree_hal_element_type_t element_type, iree_host_size_t buffer_capacity,
     char* buffer, iree_host_size_t* out_buffer_length) {
+  const char* specialname;
```

Comment:
always default initialize (don't rely on code 10+ lines away)

Reply:
i know! i was trying to imitate local style (see `const char* prefix;` below in this function).